src/Views/UserSettingsView.vala

```diff
@@ -312,17 +304,31 @@ namespace SwitchboardPlugUserAccounts.Widgets {
 
             var size_group = new Gtk.SizeGroup (HORIZONTAL);
             size_group.add_widget (header_area);
-            size_group.add_widget (content_area);
+            size_group.add_widget (content_grid);
 
             var scrolled = new Gtk.ScrolledWindow () {
-                child = content_area,
+                child = content_grid,
                 hscrollbar_policy = NEVER
             };
 
+            var info_scroll_box = new Gtk.Box (Gtk.Orientation.VERTICAL, 12) {
+                vexpand = true,
+                hexpand = true
+            };
+            info_scroll_box.append (infobar);
+            info_scroll_box.append (scrolled);
```

Comment:
The ScrolledWindow needs to be outside of the clamp otherwise you get this scrollbar in the middle of the window:

![Screenshot from 2024-10-09 15 09 20](https://github.com/user-attachments/assets/92e25029-a60b-47c5-845f-8a057ca9394d)

Reply:
Good eye, I will make the adjustment right away.